### PR TITLE
Final blog GSoC 2022

### PR DESCRIPTION
**Pull request by Harshil-Jani**
Adding the Final Version of the GSOC 2022 Blog for  Geant4 Performance Analysis and Data Vizualization Project.

### CODE REVIEW

File: _gsocblogs/2022/blog_Geant4_HarshilJani.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+---
+project: Geant4
+title: Community Bonding Period
```
Comment (agheata) [inline _gsocblogs/2022/blog_Geant4_HarshilJani.md:3]:
```suggestion
title: Geant4 - Performance Data Visualization using d3.js 
```

**Review by agheata — COMMENTED** [review]
@Harshil-Jani the blog renders fine, except that the figures are too big, maybe try to scale them to the page width. I need an OK from one of your mentors before approving it.

**Comment by agheata** [comment]
Otherwise, you have an unclean local merge mixing in commits that are not yours, I'm afraid you'll have to fetch the master, branch again, and make another PR.

**Review by agheata — CHANGES_REQUESTED** [review]
@Harshil-Jani please rebase your branch against the curret master

**Comment by amadio** [comment]
Hi @Harshil-Jani Overall the contents looks ok, but there are still merge conflicts between your branch and upstream's master branch. You need to use `git reabse` to sync your branch with latest commits from upstream. The figures seem too big for the text, and there are capitalized words in the middle of sentences that need to be fixed. Please try to pass the text through a proofreading software to help catch other typos as well. Then you can push again and tag me to go through a next round of review.

**Comment by agheata** [comment]
@amadio is this OK with you? Capitalization seems to be more a matter of style, I have no problem with that. The page and figures render fine now.

**Review by agheata — APPROVED** [review]

**Pull request merged**